community/governance.md

```diff
@@ -0,0 +1,376 @@
+so bad situations are handled quickly. The lock must be justified in the
+thread itself with a text explaining the reasons for locking and how the
+participants can contest it.
+
+- Standard
+- No need for voting to lock a thread
+
+------------------------------------------------------------------------
+
+Block a Contributor  
+In extreme cases, such as repeated harassment, it may become necessary
+to block a user completely from participating in all conda-forge
+activities. This should not be done lightly, but it may be necessary to
+do so expediently. Shorter voting periods (such as 24 hrs) are to be
+expected. The proposer of the block must provide ample justification as
+to why this is needed.
+
+- Sensitive
+- 60% Majority to pass
+- Quorum rules: standard or time-out
+
+------------------------------------------------------------------------
+
+Remove member of staged-recipes  
+The proposer must provide a justification as to why the member of staged
+recipes should be removed.
+
+- Sensitive
+- 66.7% Majority to pass
+- Quorum rules: standard or time-out
+
+------------------------------------------------------------------------
+
+Remove member of core  
+The proposer must provide an overwhelming justification as to why the
+member core should be removed.
+
+- Sensitive
+- 75% Majority to pass
+- Quorum rules: standard or time-out
+
+------------------------------------------------------------------------
+
+Overall workflow and packaging policies  
+The proposer can choose to create a poll with an external tool or call
+for voting on the GH issue in question. The voting period must be open
+for at least one core member meeting cycle to allow for clarification
+questions and discussions. Friendly reminders to vote are encouraged.
+
+- Standard
+- 50% Majority to pass
+- Quorum rules: standard, accelerated, or time-out
+
+------------------------------------------------------------------------
+
+Spending of funds  
+Proposers must specify the purpose, time limit, and source of funds that
+are to be spent. Purpose and time limit should be general enough in
+order to prevent excessive voting. For example, recurrent items (such
+as CI) should not need to be voted on each and every month. Instead,
+they should exist for a defined period of time (e.g. until the current
+migration ends, or for the next year). For such recurring expenses, the
+person coordinating spending the funds can choose to cancel the spending
+if it is deemed no longer necessary or cost-effective without calling
+another vote, although they should make reasonable efforts to notify the
+rest of core before doing so.
+
+- Standard
+- 50% Majority to pass
+- Quorum rules: standard or time-out
+
+------------------------------------------------------------------------
+
+Modifying the governance document  
+The voting should happen in the PR in question and there must be a call
+to <span class="title-ref">@conda-forge/core</span>. The voting period
```

Comment:
This seems to be a left over from converstion. Link to `/community/join/#core` I guess?

Reply:
Sounds good

Reply:
The `<span>` stuff is still here.